src/components/DivElement.js

```diff
@@ -1,6 +1,14 @@
 const HTMLElement = require('./HTMLElement');
 
 // Define class here
+class DivElement extends HTMLElement {
+  constructor(content) {
+    super(content);
+    this.content = content;
+  }
+  render() {
```

Comment:
One of the requirements for this class is to inherit the render function from the `HTMLElement` class. You can take advantage of the `super` keyword in order to pass the `tag` and `content` to the `HTMLElement` constructor. Delete this render function and see if you can figure out how to get it working that way.